hrjobcontract/CRM/Hrjobcontract/BAO/HRJobContractRevision.php

```diff
@@ -53,6 +53,32 @@ public static function create($params) {
   }
 
   /**
+   * Builds and executes a query that return all the details of a given revision
+   *
+   * Note: as there could be multiple leave records, the leave data must be
+   * fetched and processed separately
+   *
+   * @param  Array $revision
```

Comment:
can you please add a description to the $revision param saying what is the expected format for the array? I had to read all the code to figure that out